_posts/2018-03-15-integrating-redux-and-socketio.markdown

```diff
@@ -0,0 +1,118 @@
+---
+layout: post_longform
+title: "Integrating Redux and Socket.io"
+date: 2018-03-15
+categories: [Technology, positron, javascript]
+author: [luc]
+---
+
+On the engineering team at Artsy, we've built a CMS for both internal and external editors to write and publish articles. We have a team of (x) in-house editors creating new content on a daily basis. As multiple people would use the app simultaneously, something became apparent. Editors would unintentionally go and override each other’s work because there was no way to tell if another editor was currently editing an article. As a workaround, our editors are forced to edit drafts in another editor such as google docs andonly copy their work over once ready. This makes for a lackluster collaborative experience.
```

Comment:
| As a workaround, our editors are forced to edit drafts in another editor such as google docs andonly copy their work over once ready.

Actually the real workaround is that they had to let each other know in Slack anytime someone was going into a draft. I bet the number of times people typed "i'm in this article"/"i'm out" was huge compared to now 😄. The Google docs use was so editors could have the ability to comment -- something Positron doesn't support atm.

Reply:
Thinking again, the collaborative editing was def a feature that made people use Google Docs too, I think it'd be fine to include both or just keep what you have 👍

Reply:
Maybe editors should be working in Github; this seems to be working pretty well 😄

Reply:
Lol perfect, and they can push that markdown to Force and we can remove Positron entirely 😈

Reply:
ok done, switch to github for editing ✅😂